contracts/libraries/GnosisAuction.sol

```diff
@@ -103,6 +118,58 @@ library GnosisAuction {
         );
     }
 
+    function placeBid(BidDetails memory bidDetails)
+        internal
+        returns (uint256 oTokensToBuy, uint256 bidAmount)
+    {
+        // calculate how much to allocate
+        bidAmount = bidDetails
+            .lockedBalance
+            .mul(bidDetails.optionAllocationPct)
+            .div(100 * 10**2);
+        oTokensToBuy = 0; //bidAmount.div(PREMIUM)
```

Comment:
still TBD what premium we are willing to give per oToken. 

should we offer the floor premium of the auction?